crescita-personale/benefit.md

```diff
@@ -1,32 +1,40 @@
+  - [sleeve Tucano Second Skin Elements](https://www.amazon.it/dp/B009F8ARNS/ref=pe_3310731_189395851_TE_dp_1); 
+  - [Apple Magic Keyboard](https://www.apple.com/it/shop/product/MLA22T/A/magic-keyboard-italiano) (con layout a scelta tra US e IT);
+  - mouse a scelta tra [Apple Magic Mouse](https://www.apple.com/it/shop/product/MRME2Z/A/magic-mouse-2-grigio-siderale) e [Logitech MX Master 2S](https://www.logitech.com/it-it/product/mx-master-2s-flow);
+  - cuffie a cancellazione attiva del rumore (a scelta tra [Bose QuietComfort 35 II](https://www.bose.it/it_it/products/headphones/over_ear_headphones/quietcomfort-35-wireless-ii.html) e [Sony WH-1000XM3](https://www.sony.it/electronics/cuffie-archetto/wh-1000xm3));
+  - [monitor LG 27UK850-W](https://www.lg.com/it/monitor/lg-27UK850-W) (IPS, 27", 4K);
+  - [scrivania BEKANT](https://www.ikea.com/it/it/catalog/products/S69282592/#/S19022808) (delle dimensioni desiderate) e [sedia da ufficio JÄRVFJÄLLET](https://www.ikea.com/it/it/catalog/products/S99275632/#/S89275623).
+- **[Mentoring](https://github.com/nebulab/playbook/blob/master/crescita-personale/mentoring-e-1-1.md).**
+  Quando entri a far parte di Nebulab, ti sarà assegnato un mentore che è il tuo punto di 
+  riferimento non solo per tutto quello che riguarda l'azienda, ma anche per la tua crescita 
+  personale e professionale. Se sei interessato a un argomento e vorresti approfondirlo, ricorda che
+  il tuo mentore è lì per questo!
+- **[Study Friday](https://github.com/nebulab/playbook/blob/master/crescita-personale/venerdi.md).** 
+  Il venerdì è un momento per staccare dal lavoro per i clienti e concentrarci su progetti di studio 
+  e crescita personale (o aziendale). Se non sapete bene su che cosa lavorare, chiedete al vostro
+  mentore di aiutarvi a trovare un progetto adeguato al vostro percorso di crescita.
+- **[Orario flessibile](https://github.com/nebulab/playbook/blob/master/come-lavoriamo/dove-quando-e-come.md).** 
+  Guardiamo i risultati, non le ore di lavoro. Tutti nel team hanno diritto a lavorare in orari 
+  flessibili, compatibilmente con le esigenze dei clienti e degli altri membri del team. Se sei in 
+  dubbio, chiedi al cliente e ai tuoi colleghi.
+- **[Lavoro da remoto](https://github.com/nebulab/playbook/blob/master/come-lavoriamo/dove-quando-e-come.md#lavorare-da-remoto).** 
+  Nonostante abbiamo due uffici, la nostra cultura è assolutamente _remote-first_. È possibile in 
+  qualunque momento optare di lavorare da remoto, non importa se lo si vuole fare per un giorno, per 
+  un anno o a tempo indefinito.
+- **Settimane Nebulose.** Ogni anno, organizziamo almeno due Settimane Nebulose, dei team retreat di 
+  durata settimanale in cui si lavora e ci si diverte insieme. Questi incontri hanno l'obiettivo di 
+  far socializzare membri del team che lavorano in uffici diversi o da remoto; siete incoraggiati a 
+  partecipare!
```

Comment:
Non sono totalmente d'accordo. Credo che le attività di team-building servano anche ai membri del team che lavorano fisicamente nello stesso ufficio.
In generale approfondirei leggermente il discorso del team-building (motivazione, unione, condivisione) visto che è un aspetto che ci sta piuttosto a cuore.